### PR TITLE
Feature-#64: Add links to PR titles

### LINKED ISSUE #64: [Feature] Add links to pull request titles
**Is your feature request related to a problem? Please describe.**
I would like to open a pull request by selecting the pull request title in a table listed under `Activity`.

image

**Describe the solution you'd like**
When I click on `Bump y18n from 3.2.1 to 3.2.2 in /services/client` it should open https://github.com/apoclyps/my-dev-space/pull/801" in a browser.

image

**Additional context**

The title is set here for each pull request
https://github.com/apoclyps/reviews/blob/fbd8b2364ac63b3897d2c70fd2bde08739abfeff/reviews/layout/helpers.py#L42

an example of where the `link` has been used before can be seen at https://github.com/apoclyps/reviews/blob/fbd8b2364ac63b3897d2c70fd2bde08739abfeff/reviews/layout/helpers.py#L73

documentation on links is available at https://rich.readthedocs.io/en/latest/markup.html?highlight=link#links

### PR DESCRIPTION
**Pull request by yushao2**
### What's Changed
Closes #64 

### Technical Description

- Edittted the function signature of ```render_pull_request_table``` such that org/repo information is available such that the 
link to PR can be formed.
- Added link to pr using f-strings

(It's my first time contributing to a project, and during testing I can't seem to be able to click onto anything from within the terminal -- is there a specific terminal that I need to use?)

### CODE REVIEW

**Comment by apoclyps** [comment]
First of all, thank you for your contribution @yushao2!

As for terminals, I have been using iTerm2 on mac and Windows Terminal (with WSL2 on Windows) but I'm not aware of any issues with othersterminals. Which terminal are you using?

File: reviews/layout/helpers.py [hunk 1]
```diff
@@ -13,7 +13,8 @@
 from reviews.datasource import PullRequest
 
 
-def render_pull_request_table(title: str, pull_requests: List[PullRequest]) -> Table:
+def render_pull_request_table(title: str, pull_requests: List[PullRequest],
```
Comment (apoclyps) [inline reviews/layout/helpers.py:16]: You will need to run linting for the code in both files to conform to [black](https://black.readthedocs.io/en/stable/) and pass CI.

more information on running linting can be found here: https://github.com/apoclyps/reviews#linting

Comment (yushao2) [reply reviews/layout/helpers.py:16]: I'm not all too familiar with docker, but I'll try to get that set up and running and see if I could resolve that -- I'm currently just running off the terminal from my linux distro (Manjaro). Let me see if I could get a few alternatives running and see If I could click on the links.

Thanks for your help!

Comment (apoclyps) [reply reviews/layout/helpers.py:16]: The following are the dependencies you will need for linting: you can install them from the dev requirements file

```bash
pip install -r requirements_dev.txt
```

or install the individual versions needed in your virtual environment

```bash
pip install black==21.4b2
pip install flake8==3.9.1
pip install flake8-unused-arguments==0.0.6
pip install freezegun==1.1.0
pip install isort==5.8.0
pip install mypy==0.812
pip install pylint==2.8.2
pip install pytest==6.2.3
pip install pytest-cov==2.11.1
```

Once installed, the commands you need to run can be found in the `docker-compose.yml` file: https://github.com/apoclyps/reviews/blob/main/docker-compose.yml

The following is a list of commands used when linting ( each of these are required to pass before CI will pass).

```bash
isort .
black --line-length 88 --check .
flake8 .
mypy . --ignore-missing-imports
pylint --rcfile=.pylintrc reviews
pytest -cov=reviews -cov-report -vvv
```

**Comment by apoclyps** [comment]
@yushao2 I can confirm the link works for me when using `control + click to open the link:

image

**Comment by yushao2** [comment]
> @yushao2 I can confirm the link works for me (on Windows Terminal) when using `control + click to open the link:
> 
> image

Unable to get docker working right, but managed to perform the linting steps anyway -- this PR should be good for merger if it's all good with you :)

**Comment by apoclyps** [comment]
Looks good. Thank you once again for your contribution :)

**Pull request merged**